.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug Report
+about: Report a bug or unexpected behavior
+title: ""
```

Comment:
```suggestion
title: "bug: ..."
```

Reply:
We don't name our issues with `bug`, but rather with the relevant tag. This contradicts the naming so far? Also, we seem to be using the PR title naming for bugs/issues for some reason, but I don't expect external contributors/users to know about this.

Reply:
OK good point